ext/fetch/26_fetch.js

```diff
@@ -77,6 +77,7 @@
       hasBody,
       bodyLength,
       body,
+      isRelative
```

Comment:
Sorry, just noticed another missing trailing comma 🙈 
```suggestion
      isRelative,
```